<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>527</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
    <pointsize>12</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(60, 60, 60);
color:  rgb(155, 155, 155);
border-radius: 5%;
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
  </property>
  <property name="locale">
   <locale language="Ukrainian" country="Ukraine"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="placeHolderChoice">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>180</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
	background-color: rgb(80, 80, 80);
	color:  rgb(125, 125, 125);
	border-radius: 5%;
}

QToolButton:hover {
	background-color: rgb(120, 120, 120);
	color:  rgb(255, 255, 255);
	border-radius: 5%;
	transition: all 0.3s ease 0s;
}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadAudio">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(80, 80, 80);
	font: 75 12pt &quot;MS Shell Dlg 2&quot;;
	color:  rgb(155, 155, 155);
	border-radius: 5%;
}

QPushButton:hover {
	background-color: rgb(120, 120, 120);
	font: 75 12pt &quot;MS Shell Dlg 2&quot;;
	color:  rgb(255, 255, 255);
	border-radius: 5%;
	transition: all 0.3s ease 0s;
}</string>
    </property>
    <property name="text">
     <string>Завантажити Аудіо</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadVideo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(80, 80, 80);
	font: 75 12pt &quot;MS Shell Dlg 2&quot;;
	color:  rgb(155, 155, 155);
	border-radius: 5%;
}

QPushButton:hover {
	background-color: rgb(120, 120, 120);
	font: 75 12pt &quot;MS Shell Dlg 2&quot;;
	color:  rgb(255, 255, 255);
	border-radius: 5%;
	transition: all 0.3s ease 0s;
}</string>
    </property>
    <property name="text">
     <string>Завантажити Відео</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="placeHolder">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser {
	background-color: rgb(80, 80, 80);
	color:  rgb(155, 155, 155);
	border-radius: 5%;
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	padding-top: 2px;
}

</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Вкажіть шлях, де зберегти файл&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="linkYouTube">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(80, 80, 80);
	font: 75 12pt &quot;MS Shell Dlg 2&quot;;
	color:  rgb(155, 155, 155);
	border-radius: 5%;
}

QPushButton:hover {
	background-color: rgb(120, 120, 120);
	font: 75 12pt &quot;MS Shell Dlg 2&quot;;
	color:  rgb(255, 255, 255);
	border-radius: 5%;
	transition: all 0.3s ease 0s;
}</string>
    </property>
    <property name="text">
     <string>Вказати посилання для завантаження</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLogo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>YTDlogo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="labelText">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>18</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 75 18pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
}
</string>
    </property>
    <property name="text">
     <string>YouTube Downloader</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>10</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
	background-color: rgb(90, 90, 90);
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
	color: white;
	border-radius: 5%;
	text-align: center;
}

QProgressBar::chunk {
	background-color: ;
	background-color:qlineargradient(spread:pad, x1:0.136, y1:0.0905455, x2:0.966, y2:0.0625, stop:0 rgba(108, 255, 168, 245), stop:0.943182 rgba(0, 212, 41, 241));
	border-radius: 5%;
}</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
